### ISSUE TITLE
ConfigParser fails during parsing of /etc/preupgrade-assistant.conf

### ISSUE THREAD

**Issue by pirat89** [phracek/preupgrade-assistant#73]
It's due to different style of section [home_directories], which is not really INI type.

```python
Traceback (most recent call last):
  File "<some content>", line 8, in ?
    from preup.script_api import *
  File "/usr/lib/python2.4/site-packages/preup/script_api.py", line 544, in ?
    load_pa_configuration()
  File "/usr/lib/python2.4/site-packages/preup/script_api.py", line 508, in load_pa_configuration
    config.read(PREUPGRADE_CONFIG)
  File "/usr/lib64/python2.4/ConfigParser.py", line 267, in read
    self._read(fp, filename)
  File "/usr/lib64/python2.4/ConfigParser.py", line 490, in _read
    raise e
ConfigParser.ParsingError: File contains parsing errors: /etc/preupgrade-assistant.conf
	[line 12]: '.config/\n'
	[line 13]: '.mozilla/'
```

**Comment by phracek** [comment]
Yeah, I know. In order to support rhel-5 I suggest to use dirs=./config,./mozilla.

Separated by comma.

**Comment by pirat89** [comment]
Resolved by commit 0ee1f11
